tests/system/action/meeting/test_clone.py

```diff
@@ -118,6 +123,30 @@ def test_clone_with_users(self) -> None:
         self.assert_model_exists("meeting/1", {"user_ids": [1]})
         self.assert_model_exists("meeting/2", {"user_ids": [1]})
 
+    def test_clone_with_set_fields(self) -> None:
+        self.set_models(self.test_models)
+
+        response = self.request(
+            "meeting.clone",
```

Comment:
Please check also for start_time and end_time

Reply:
Add a test for start_time and end_time.